src/content/guides/partial-payment-extension.mdx

```diff
@@ -7,8 +7,17 @@ nav:
 ---
 
 Partial Payment is an extension to the Payment Request Protocol that allows
-customers to pay a partial amount of a payment request using Centrapay.
-However, it is the responsibility of the integrating POS to ensure payment of the remaining amount.
+customers to utilize different forms of payment. This allows Centrapay Payments
+to easily complement other methods of payment. In addition
+leaveraging this with payment methods such as
+[Tokens](https://docs.centrapay.com/api/assets#tokens-experimental) will allow you
```

Comment:
This guide is geared towards POS and Merchants integrating the partial payment extension.
We don't need to make them aware of it as it's not relevant to their integration.

Reply:
When we had the discussion we thought that it could benefit from a bit more of an introduction in the absence of a marketing website

Reply:
This has been run past Matthias and updated.